ansible_runner/interface.py

```diff
@@ -34,7 +34,8 @@
 from ansible_runner.utils import (
     dump_artifacts,
     check_isolation_executable_installed,
-    santize_json_response
+    santize_json_response,
+    signal_handler
```

Comment:
Trailing commas make future diffs easier to read.
```suggestion
    signal_handler,
```

Reply:
✅